Code/Framework/AzToolsFramework/Tests/EditorTransformComponentSelectionTests.cpp

```diff
@@ -2973,4 +2973,176 @@ namespace UnitTest
         EXPECT_THAT(hoveredEntityIdAccentRemoved, IsTrue());
         EXPECT_THAT(hoveredEntityEntityId, Eq(AZ::EntityId(12345)));
     }
+
+    class EditorTransformComponentSelectionRenderGeometryIntersectionFixture : public ToolsApplicationFixture
+    {
+    public:
+        void SetUpEditorFixtureImpl() override
+        {
+            auto* app = GetApplication();
+            // register a simple component implementing BoundsRequestBus and EditorComponentSelectionRequestsBus
+            app->RegisterComponentDescriptor(BoundsTestComponent::CreateDescriptor());
+            // register a component implementing RenderGeometry::IntersectionRequestBus
+            app->RegisterComponentDescriptor(RenderGeometryIntersectionTestComponent::CreateDescriptor());
+
+            auto createEntityWithGeometryIntersectionFn = [](const char* entityName)
+            {
+                AZ::Entity* entity = nullptr;
+                AZ::EntityId entityId = CreateDefaultEditorEntity(entityName, &entity);
+
+                entity->Deactivate();
+                entity->CreateComponent<RenderGeometryIntersectionTestComponent>();
+                entity->Activate();
+
+                return entityId;
+            };
+
+            m_entityIdGround = createEntityWithGeometryIntersectionFn("Entity1");
+            m_entityIdBox = createEntityWithGeometryIntersectionFn("Entity2");
+
+            if (auto ground = AzToolsFramework::GetEntityById(m_entityIdGround)->FindComponent<RenderGeometryIntersectionTestComponent>())
```

Comment:
Sure I can add it, but I actually don't think it's in the coding standards (maybe you should raise this with sig-core? otherwise it's difficult to recommend on PR reviews)